src/provider/java/azulJavaVersions.ts

```diff
@@ -0,0 +1,21 @@
+import { AZUL_API } from "#common/constants/urls.ts";
+import { defineProvider } from "#core/provider.ts";
+import { AzulJavaPackages, AzulJavaVersions } from "#schema/java/azulJavaData.ts";
+
+const RUNTIMES_URL = new URL("v1/", AZUL_API);
+
+export default defineProvider({
+	id: "azul-java",
+
+	async provide(http): Promise<AzulJavaPackages[]> {
+		const versions = AzulJavaVersions.parse(
+			(await http.getCached(new URL("zulu/packages?availability=GA&latest=true&os=windows&arch=x64&archive_type=zip&javafx_bundled=false&java_package_type=jre", RUNTIMES_URL), "azul-java-windows-versions.json")).json()
+		);
```

Comment:
```suggestion
		const versions = AzulJavaVersions.parse(
			(await http.getCached(new URL("zulu/packages?availability=GA&latest=true&os=windows&arch=x64&archive_type=zip&javafx_bundled=false&java_package_type=jre", RUNTIMES_URL), "java-windows-versions.json")).json()
		);
```
...why just windows though